modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -124,27 +125,23 @@ Save the output. You will use it in the next steps.
 +
 [source,terminal]
 ----
-$ aws iam create-policy \
+POLICY=$(aws iam create-policy \
```

Comment:
Can we use POLICY_ARN here and use $POLICY_ARN in L132?

Reply:
Agree, POLICY_ARN is better.

Reply:
Updated, POLICY_ARN in Line 128 and $POLICY_ARN in Line 132.